### ISSUE TITLE
Upload from terminal

### ISSUE THREAD

**Issue by sicenul** [AlphaNecron/Chakram#52]
Hello, is it possible to upload using terminal? using curl maybe? How to do it?
Thanks.

**Comment by AlphaNecron** [comment]
Yes, you can upload with terminal, refer to here: https://zipline.diced.tech/docs/guides/uploaders/curl

**Comment by sicenul** [comment]
Thanks, it work.
But the original filename is not saved, right? Is it possible to add the original filename?
![image](https://user-images.githubusercontent.com/64000634/199370131-94227452-63db-4d3c-8848-1bcd0be57efb.png)
FYI, I use docker-compose with v0 version
